.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+me: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
I think CI did not run on this branch because this says `main` but the branch is called `master`? So let's rename one of them

Reply:
Done in bab4ade

Reply:
Huh, why is CI still not running on pushes to this branch? Is it disabled in repo settings or something?

Reply:
Found the issue 😅 6b40870